### PR TITLE
style: fixed theming for buttons.

### PR DESCRIPTION
**Pull request by zilaei**
Tried to not change to much other then using our defined theme for coloring.
Removed custom colored buttons from stories, uncertain if these should be allowed.

### CODE REVIEW

File: source/components/atoms/Button/Button.stories.js [hunk 1]
```diff
@@ -48,39 +48,39 @@ storiesOf('Button', module)
...
+  //   <StoryWrapper {...props}>
+  //     <FlexContainer>
+  //       <Flex>
+  //         <CustomButton colorSchema="light" block>
+  //           <CustomButtonText>Button with custom styles</CustomButtonText>
+  //         </CustomButton>
+  //       </Flex>
+  //     </FlexContainer>
+  //   </StoryWrapper>
+  // ))
```
Comment (dannil76) [inline source/components/atoms/Button/Button.stories.js:61]: Please remove unused code. Tip, save it in a gist. :)

**Review by dannil76 — CHANGES_REQUESTED** [review]
Minor

**Comment by TeddyAndersson** [comment]
Made some changes in addition to your changes @zilaei, hope you don't mind. The button should now be more in line with the figma design :)

**Comment by TeddyAndersson** [comment]
However we have some additions that we need to make, and i think this is a good case for defining a button object in the theme object. But i do not think these changes is suited for this PR but rather another task. Since it's new styling to the button in general :)

**Review by TeddyAndersson — APPROVED** [review]
Looks good to me, good if someone can look over the last changes that i made :)

**Review by dannil76 — APPROVED** [review]

**Comment by zilaei** [comment]
@TeddyAndersson: Changes welcomed and accepted! 👍

**Pull request merged**